src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -1764,6 +1764,10 @@
   </Node>
 
   <Node Name="BoundStackAllocArrayCreation" Base="BoundExpression">
```

Comment:
>BoundStackAllocArrayCreation [](start = 14, length = 28)

This isn't primarily related to the change in this PR, but it looks like this node doesn't follow the guideline that only leaf nodes have kinds.

Reply:
Fixing it and adding that to the bound node design document in the next iteration.

---
In reply to: [359482457](https://github.com/dotnet/roslyn/pull/40457#discussion_r359482457) [](ancestors = 359482457)